webpack.common.js

```diff
@@ -27,7 +27,7 @@ const targets /*: Array<Target> */ = [
     },
     {
         name: 'contrib/copy-tex',
-        entry: './contrib/copy-tex/copy-tex.js',
+        entry: './contrib/copy-tex/copy-tex.webpack.js',
```

Comment:
If we're using `rollup.config.js` to build all of these targets, why is `copy-tex` the only one that needed to be changed to use `.webpack.js`?

Reply:
@kevinbarabash `copy-tex` imports a CSS module that ESM doesn’t support. So we use separate entry point for webpack.